scripts/test-verify-phase-52-7-1-namespace-layout-inventory-contract.sh

```
@@ -0,0 +1,135 @@
+  "${missing_proposed_namespace_repo}/${contract_path}"
+assert_fails_with \
+  "${missing_proposed_namespace_repo}" \
+  "Missing Phase 52.7.1 namespace inventory row: proposed canonical namespace"
+
+wrong_proposed_namespace_repo="${workdir}/wrong-proposed-namespace"
+create_valid_repo "${wrong_proposed_namespace_repo}"
+perl -0pi -e 's/aegisops\.control_plane/aegisops.controlplane/g' \
+  "${wrong_proposed_namespace_repo}/${contract_path}"
+assert_fails_with \
+  "${wrong_proposed_namespace_repo}" \
+  'Missing Phase 52.7.1 namespace/layout statement: The proposed canonical namespace `aegisops.control_plane` is a future target only.'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)